worker/worker.go

```diff
@@ -662,21 +660,27 @@ func (w *worker) slabMigrateHandler(jc jape.Context) {
 		return
 	}
 
+	// decode the contract set from the query string
+	var contractset string
+	if jc.DecodeForm(api.QueryStringParamContractSet, &contractset) != nil {
+		return
+	} else if contractset != "" {
```

Comment:
I would make specifying the contract set for migrations mandatory. Imo migrating is too important to not be explicit about it. e.g. if you change the contract set in the autopilot but forget to update the default contract set you are in a world of pain. 
So the autopilot should be forced to explicitly provide its current set.

Reply:
Great idea!